_episodes/04-programming.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Effective Programming"
+teaching: 0
+exercises: 0
+questions:
+- "Why should I care about code quality?"
+- "How do I make my code more legible?"
+objectives:
+- "Create expressive variable names."
+- "Write effective code comments."
+- "Write code that is easy to read."
+- "Use temporary variables, scalars, and files."
+- "Understand and use local and global macros."
+- "Reuse the results of other commands."
+- "Automate repetitive tasks using `foreach` and `forvalues`."
+keypoints:
+- "Write expressive variable names."
+- "Comment the why, not the what."
+- "Use for loops to automate anything that happens more than twice."
```

Comment:
Should `for` be in backticks?

Reply:
No, because there are two separate, `foreach` and `forvalues`.